bundles/org.openhab.binding.modbus.lambda/README.md

```diff
@@ -0,0 +1,174 @@
+# Lambda Heat Pump
+
+This extension adds support for the Lambda Heat Pump modbus protocol as provided by
+https://lambda-wp.at/wp-content/uploads/2022/01/Modbus-Protokoll-und-Beschreibung.pdf 
+
+A Lambda Heat Pump has to be reachable within your network. 
+If you plan to use the E-Manager part to hand over your PV excess to the heat pump ask Lambda support to 
+configure it to  
+E-Meter Kommunikationsart:          ModBus Client
+E-Meter Messpunkt:                  E-Eintrag
+
+Other configurations of the E-Manager are not supported (yet).
+
+Up to now only the following configuration is supported:
+- only one heatpump(heatpump1)
+- only one buffer for the heating (buffer1)
+- only one buffer for the water heating (boiler1)
+
+
+## Supported Things
+
+This bundle adds the following thing types to the Modbus binding.
+Note, that the things will show up under the Modbus binding.
+
+| Thing              | ThingTypeID         | Description                                         |
+| ------------------ | --------------------| --------------------------------------------------- |
+| Lambda Heat Pump   | lambda-heatpump     | A lambda heat pump visible in the local network     |
+
+## Discovery
+
+This extension does not support autodiscovery. The things need to be added manually.
+
+A typical bridge configuration would look like this:
+
+```java
+Bridge modbus:tcp:bridge [ host="10.0.0.2", port=502, id=1 ]
+```
+
+## Thing Configuration
+
+You first need to set up a TCP Modbus bridge according to the Modbus documentation.
+You then add the lambda heat pump as part of the modbus binding.
+Things in this extension will use the selected bridge to connect to the device.
+
+The following parameters are valid for all thing types:
+
+| Parameter | Type    | Required | Default if omitted | Description                                                                |
+| --------- | ------- | -------- | ------------------ | -------------------------------------------------------------------------- |
+| refresh   | integer | no       | 30                 | Poll interval in seconds. Increase this if you encounter connection errors |
+| maxTries  | integer | no       | 3                  | Number of retries when before giving up reading from this thing.           |
+
+## Channels
+
+Channels are grouped into channel groups.
+
+### General Ambient Group
+
+This group contains general operational information about the heat pump.
+
+
+| Channel ID                       | Item Type          | Read only | Unit     | Description                                                              |
```

Comment:
The Unit column can be omitted as the Item Type already describes the type expected. The Unit however is based (or should be) based on the user configured measurement system. (C vs F)